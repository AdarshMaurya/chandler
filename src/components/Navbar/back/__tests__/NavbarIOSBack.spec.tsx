import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import {NavbarIOSBack} from '../NavbarIOSBack';

describe('Components', () => {
  describe('NavbarIOSBack', () => {
    const onPress = jest.fn();
    it('should renders correctly', () => {
      expect(
        toJson(shallow(<NavbarIOSBack onPress={onPress} />))
      ).toMatchSnapshot();
    });
  });
});
