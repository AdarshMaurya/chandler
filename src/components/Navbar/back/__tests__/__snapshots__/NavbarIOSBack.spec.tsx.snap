// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components NavbarIOSBack should renders correctly 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  hitSlop={
    Object {
      "bottom": 15,
      "left": 15,
      "right": 15,
      "top": 15,
    }
  }
  onPress={[MockFunction]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "paddingLeft": 12,
    }
  }
>
  <Icon
    allowFontScaling={false}
    color="white"
    name="ios-arrow-back"
    size={30}
  />
</TouchableOpacity>
`;
